src/schemas/application.ts

```
@@ -118,4 +118,93 @@
+    .string()
+    .min(1, "An answer is required for this question")
+    .refine((value) => value.split(/\s/g).length <= 150, {
+      message: "Must be less than 150 words",
+    }),
+  socialText: z.array(z.string()).default([]),
+  interests: z
+    .string()
+    .refine((value) => value.split(/\s/g).length <= 150, {
+      message: "Must be less than 150 words",
+    })
+    .transform((string) => (!!string ? string : null))
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)